packages/next/src/server/app-render/required-scripts.tsx

```diff
@@ -0,0 +1,56 @@
+import type { BuildManifest } from '../get-page-files'
+
+import ReactDOM from 'react-dom'
+
+export function getRequiredScripts(
+  buildManifest: BuildManifest,
+  assetPrefix: string,
+  SRIManifest: undefined | Record<string, string>,
+  qs: string
+): [() => void, string | { src: string; integrity: string }] {
+  let preinitScripts: () => void
+  let preinitScriptCommands: string[] = []
+  let bootstrapScript: string | { src: string; integrity: string } = ''
+  const files = buildManifest.rootMainFiles
+  if (files.length === 0) {
+    throw new Error(
+      'expected to have at least one bootstrap script. This is a bug in Next.js'
```

Comment:
```suggestion
      'Invariant: missing bootstrap script. This is a bug in Next.js'
```

Lets use the "Invariant" error message like the others we currently have